src/styles/header.scss

```diff
@@ -0,0 +1,62 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  font-size: 12px;
+  align-items: center;
```

Comment:
Box-shadow for header is absent. Pay attention to the design
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/12037628-ae4f-4a9e-ad76-96d1ac008874)
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/04a3aa46-6bdc-4d7c-bbf3-011895811dd1)
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/11abadd5-f503-4c00-9a3c-f00b41f699f0)